keymap/tutorial.kbd

```diff
@@ -344,8 +349,9 @@
 
   We have already seen other examples of modded buttons, \(, \), *, and +. There
   are no Keycodes for these buttons in KMonad, but they are buttons. They simply
-  evaluate to `(around lsft x)`. All shifted numbers have their corresponding
-  characters, the same is true for all capitals, and < > : ~ " | { } + and ?.
+  evaluate to `(around lsft x)`. All shifted numbers ( ! @ # $ % ^ & * \( \) )
+  letters (A-Z), and symbols ( ~ \_ + { } | : " < > ? ) represent themselves in
+  this way.
 
```

Comment:
I think this new version is too US-centric, layout wise; other countries have different symbols on `S-<number>`